inputs.tf

```diff
@@ -67,6 +67,11 @@ variable "minimum_node_count" {
   description = "Minimum nodes for the node pool. This is the total nodes so for regional deployments it is the total nodes across all zones."
 }
 
+variable "project_id" {
+  type        = string
+  description = "Project id of the account where gke will be deployed. Used by provider"
```

Comment:
This variable is added, but never used?  Also there is already `google_project` above.